src/middleware/permissionsMiddleware.js

```diff
@@ -41,6 +41,12 @@ const isAuthor = rule({ cache: 'no_cache' })(async (parent, args, { user, driver
   return authorId === user.id
 })
 
+const fromUserMatchesCurrentUser = rule({ cache: 'no_cache' })(async (parent, args, { user, driver }) => {
+  if (!user) return false
+  const { from: { id: fromId } } = args
+  return user.id === fromId
```

Comment:
@appinteractive better than redundantly checking the `from` user is to override it with the current authenticated user. Too bad, we have to change the augmented `AddPostDisabledBy` mutation and then the generated resolver doesn't work anymore.

Reply:
I would write our own resolver/mutation here anyway like I did with the (in)follow

Reply:
Why not just adding something like that? `disable(Id: postId, type: Post)`